apps/full-site-editing/full-site-editing-plugin/editor-gutenboarding-launch/index.ts

```diff
@@ -52,12 +55,25 @@ function updateEditor() {
 
 		// Wrap 'Launch' button link to frankenflow.
 		const launchLink = document.createElement( 'a' );
-		launchLink.href = window?.calypsoifyGutenberg?.frankenflowUrl as string;
+
+		// Assert reason: We have an early return above with optional and falsy values. This should be a string.
+		const launchHref = window?.calypsoifyGutenberg?.frankenflowUrl as string;
+
+		launchLink.href = launchHref;
 		launchLink.target = '_top';
 		launchLink.className = 'editor-gutenberg-launch__launch-button components-button is-primary';
 		const textContent = document.createTextNode( __( 'Launch' ) );
 		launchLink.appendChild( textContent );
 
+		const saveAndNavigate = async ( e: Event ) => {
+			// Disable href navigation
+			e.preventDefault();
+			await dispatch( 'core/editor' ).autosave();
```

Comment:
Looks like firing `savePost()` instead of `autosave()` would help here for the prompt to not appear @sirreal

Reply:
👍  Happy to try that. I did notice after clicking `update` you won't see the message, so we should also be able to emulate whatever it's doing.

Reply:
Let's try that, seems straightforward 👍

Reply:
`savePost` works well: #43116